packages/flutter_tools/flutter_analysis_options

```diff
@@ -25,28 +27,29 @@ analyzer:
     strong_mode_down_cast_composite: ignore
     # we allow having TODOs in the code
     todo: ignore
+  exclude:
```

Comment:
(Will it exclude dart files in flutter/bin or flutter/packages/bin? If so, why do we want that?)